rocketpy/EnvironmentAnalysis.py

```diff
@@ -588,22 +594,99 @@ def calculate_maximum_wind_gust(self):
         self.max_wind_gust = np.max(self.wind_gust_list)
         return self.max_wind_gust
 
-    # TODO: Implement
+    # TODO: Create tests
     def calculate_wind_gust_distribution(self):
         """Get all values of wind gust speed (for every date and hour available)
-        and plot a single distribution. Expectedr result is a Weibull distribution.
+        and plot a single distribution. Expected result is a Weibull distribution.
         """
-        ...
+        self.wind_gust_list = [
+            dayDict[hour]["surfaceWindGust"]
+            for dayDict in self.surfaceDataDict.values()
+            for hour in dayDict.keys()
+        ]
+        plt.figure()
+        # Plot histogram
+        plt.hist(
+            self.wind_gust_list,
+            bins=int(len(self.wind_gust_list) ** 0.5),
+            density=True,
+            histtype="stepfilled",
+            alpha=0.2,
+            label="Wind Gust Speed Distribution",
+        )
+
+        # Plot weibull distribution
+        c, loc, scale = scipy.stats.weibull_min.fit(self.wind_gust_list)
```

Comment:
In my machine I'm receiving the following error:

`AttributeError: module 'scipy' has no attribute 'stats'`

Probably due to (https://stackoverflow.com/questions/22108017/cannot-use-scipy-stats), which could be solved by adding a line `import scipy.stats` at the beggining of the file